dcos_installer/upgrade.py

```diff
@@ -51,18 +51,27 @@
 SKIP_CHECKS=false
 VERBOSE=false
 
-if [[ $# -ne 0 ]]; then
-    for var in "$@"; do
-        if [[ "$var" = "--skip-checks" ]]; then
-            echo "Skipping checks"
-            SKIP_CHECKS=true
-        fi
-        if [[ "$var" = "--verbose" ]]; then
-            echo "Verbose mode on"
-            VERBOSE=true
-        fi
-    done
-fi
+while (( "$#" )); do
+  case "$1" in
+    --skip-checks)
+      shift
+      echo "Skipping checks"
+      SKIP_CHECKS=true
+      ;;
+    -v|--verbose)
+      shift
+      echo "Verbose mode on"
+      VERBOSE=true
+      ;;
+    -*|--*=) # unsupported flags
```

Comment:
I'm curious why the standard `*)` to catch everything else doesn't work here. I haven't used the `while (( "$#" ));` approach before.